### PR TITLE
Update the binaries and release notes process

### PR DESCRIPTION
**Pull request by mangelajo**
The binaries are now automatically generated, and we explain how to handle the
release notes.

### CODE REVIEW

File: src/content/contributing/release-process/_index.en.md [hunk 1]
```diff
@@ -188,19 +188,10 @@ At this point the images tagged with 0.2.0-rc0 will be available [here](https://
...
-git fetch -a -v -t upstream
-git checkout v0.2.0-rc0
-rm bin/subctl*
-make build-cross
-ls -la bin/subctl*
-```
-At this point, you should see subctl binaries generated and listed for the various platforms under bin.
-Go to https://github.com/submariner-io/submariner-operator/tags, find the tag for v0.2.0-rc0 and select "Edit release" to the right. Then upload the generated subctl binaries.
+At this point, you should see subctl binaries generated and listed for the various platforms under the release
+ https://github.com/submariner-io/submariner-operator/tags, find the tag for v0.2.0-rc0
```
Comment (tpantelis) [inline src/content/contributing/release-process/_index.en.md:194]: It seems to end mid-sentence - did you intend to remove the "and select "Edit release..." part?

Comment (Jaanki) [reply src/content/contributing/release-process/_index.en.md:194]: Fixing the tag (v0.2.0-rc0) is not a good idea. Or maybe a note saying that the tag is just for an example.

Comment (mangelajo) [reply src/content/contributing/release-process/_index.en.md:194]: Yes it was intended, all that happens automatically now. May be we should leave it to make sure we add the right comments to the release?

Comment (mangelajo) [reply src/content/contributing/release-process/_index.en.md:194]:
```suggestion
 https://github.com/submariner-io/submariner-operator/tags, find the tag for v0.2.0-rc0 , verify that the binaries uploaded, the process needs around 5 minutes.
 
 ### Update the release notes
 
 Go to https://github.com/submariner-io/submariner-operator/tags, find the tag for v0.2.0-rc0 and select "Edit release" to the right. 
 
Update the release notes.
 
If this is a pre-release, mark the checkbox "This is a pre-release".
 
 If this is not a pre-release update [the release notes on the website](https://github.com/submariner-io/submariner-website/edit/master/src/content/releases/_index.en.md).
```

**Comment by mangelajo** [comment]
@Jaanki @tpantelis @dfarrell07 can you have another eye?

I added also some details about the release notes.

File: src/content/contributing/release-process/_index.en.md [hunk 2]
```diff
@@ -191,7 +191,17 @@ At this point the images tagged with 0.2.0-rc0 will be available [here](https://
...
- https://github.com/submariner-io/submariner-operator/tags, find the tag for v0.2.0-rc0 
+ https://github.com/submariner-io/submariner-operator/tags, find the tag for v0.2.0-rc0 , verify that the binaries uploaded, the process needs around 5 minutes.
+ 
+ ### Update the release notes
+ 
+ Go to https://github.com/submariner-io/submariner-operator/tags, find the tag for v0.2.0-rc0 and select "Edit release" to the right. 
+ 
+Update the release notes.
+ 
+If this is a pre-release, mark the checkbox "This is a pre-release".
```
Comment (Jaanki) [inline src/content/contributing/release-process/_index.en.md:202]: Repeated at L206.

File: src/content/contributing/release-process/_index.en.md [hunk 3]
```diff
@@ -188,19 +188,20 @@ At this point the images tagged with 0.2.0-rc0 will be available [here](https://
...
+ 
+ If this is not a pre-release update [the release notes on the website](https://github.com/submariner-io/submariner-website/edit/master/src/content/releases/_index.en.md).
 
 If this is a pre-release, mark the checkbox "This is a pre-release".
```
Comment (mangelajo) [inline src/content/contributing/release-process/_index.en.md:206]:
```suggestion
```

File: src/content/contributing/release-process/_index.en.md (hunk 2 above)
Comment (mangelajo) [reply src/content/contributing/release-process/_index.en.md:202]: Oops, done :)

**Comment by mangelajo** [comment]
@Jaanki hopefully done now. This will need squash ':)

**Pull request merged**